Packs/SalesforceIndicators/Integrations/SalesforceIndicators/SalesforceIndicators.py

```diff
@@ -91,33 +92,36 @@ def get_object_description(self):
         return res
 
 
-def fetch_indicators_command(client, params, manual_run=False):
+def fetch_indicators_command(client, manual_run=False):
 
     indicators_unparsed = list()
     indicators = list()
     now = datetime.utcnow()
     history_date = dateparser.parse(f"{client.history} days ago", settings={'RELATIVE_BASE': now})
     assert history_date is not None, f'could not parse {client.history} days ago'
     date_filter = history_date.strftime("%Y-%m-%dT%H:%M:%S.000+0000")
-    integration_context = get_integration_context()
-    if integration_context:
-        last_run = integration_context.get('lastRun')
+    stored_last_run = demisto.getLastRun()
+
+    if stored_last_run:
+        last_run = stored_last_run.get('lastRun')
     else:
         last_run = None
```

Comment:
If `demisto.getLastRun()`has no value, it will return an empty dict, and preforming `.get('lastRun')` on it will result `None`.
```suggestion
    last_run = demisto.getLastRun().get('lastRun')
```
This change should also be added to the release notes.

Reply:
I made the code change (plus a few other bits)